src/main/java/de/tum/in/www1/artemis/web/rest/StudentExamResource.java

```diff
@@ -148,6 +148,34 @@ public StudentExamResource(ExamAccessService examAccessService, StudentExamServi
         return ResponseEntity.ok(studentExamRepository.save(studentExam));
     }
 
+    /**
+     * POST /courses/{courseId}/exams/{examId}/studentExams/submit : Submits the student exam
+     * Updates all submissions and marks student exam as submitted according to given student exam
+     * @param courseId      the course to which the student exams belong to
+     * @param examId        the exam to which the student exams belong to
+     * @param studentExam   the student exam parameter (request body)
```

Comment:
Please be more specific. It is important to note that it includes exercises, participations and submissions, otherwise the call won't work